adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
If data.timestamp should be zero for whatever reason...

Accuracy of als timestamps is not critical feature here (or really anything to do with mobile sensors). Als rarely run at a data rate that it would actually matter anyway.